x/epoch/types/query.pb.go

```diff
@@ -150,7 +150,7 @@ func (m *QueryEpochRequest) XXX_DiscardUnknown() {
 var xxx_messageInfo_QueryEpochRequest proto.InternalMessageInfo
 
 type QueryEpochResponse struct {
-	Epoch Epoch `protobuf:"bytes,11,opt,name=epoch,proto3" json:"epoch"`
+	Epoch Epoch `protobuf:"bytes,1,opt,name=epoch,proto3" json:"epoch"`
```

Comment:
why did this proto gen file have changes if there were no associated proto def changes for epoch?

Reply:
Good question, I'm I don't think we're currently checking if generated proto files are updated -- this was the output when I ran it on the whole repo